packager/media/formats/mp4/box_definitions.h

```diff
@@ -382,6 +394,13 @@ struct AudioSampleEntry : Box {
   AC4Specific dac4;
   OpusSpecific dops;
   FlacSpecific dfla;
+  Mha1Specific mhac;
+
+  //23008-3 decoder specific config
+  CodecConfiguration codec_configuration;
```

Comment:
I am a bit confused on having both `mhac` and `codec_configuration`. It seems like the fourcc for both configs `mhaC`. Do we need both?

Reply:
Shaka Packager has separate data structures for audio and video, but actually, the mpeg-h standard treats object audio like, so this is a workaround for the design limitation.